apps/test/unit/templates/manageStudents/ManageStudentsTableTest.js

```diff
@@ -184,6 +185,59 @@ describe('ManageStudentsTable', () => {
       );
     });
 
+    it('does render the gender column if loginType is secret picture', () => {
+      const wrapper = mount(
+        <Provider store={getStore()}>
+          <ManageStudentsTable />
+        </Provider>
+      );
+      expect(wrapper.find(ManageStudentsGenderCell).exists()).to.be.true;
+    });
+
+    it('does not render the gender column if loginType is secret word', () => {
```

Comment:
Should this message be `does` instead of `does not`, the same as the above test for secret picture? I would assume secret word and secret picture would have the same rendered view?

Reply:
Whoops, yeah good catch. I must have copy-pasted the messages wrong. It should be `does render` for secret word and picture, and `does not render` for email and clever/google classroom.